audio-worklet/lib/audio-worklet-helper.js

```diff
@@ -14,10 +14,16 @@ const AudioWorkletHelper = (() => {
   let reporterInitialized_ = false;
   let eReporterDiv_;
 
+  // Check if BaseAudioContext has AudioWorklet.
+  let IsAudioWorkletInBaseAudioContext_ = (() => {
+    let context = new OfflineAudioContext(1, 1, 44100);
+    return context.audioWorklet && 
+        typeof context.audioWorklet === 'AudioWorklet' &&
```

Comment:
Need to fix this because the typeof is "object", as mentioned on the slack channel.

Reply:
This PR fixes it: https://github.com/GoogleChromeLabs/web-audio-samples/pull/130